### ISSUE TITLE
ERROR (BAD_SUPVISORS_STATE: Supvisors (state=INITIALIZATION) not in state ['OPERATION'] to perform request)

### ISSUE THREAD

**Issue by remusnegrota** [julien6387/supvisors#104]
Installed the plugin following the docs and the example conf.
All works well, `supvisorsctl` command works, but if I try to do anything specific to supvisors plugin,  for example to `update_numprocs` it throws the above error

The supervisord logs the following:
```
2022-05-25 16:02:38,530;INFO;SupvisorsInstanceStatus.state: Supvisors=test-server.addpipe.com is CHECKING
2022-05-25 16:02:38,530;WARN;InitializationState.next: synchro timed out
2022-05-25 16:02:38,531;INFO;SupvisorsInstanceStatus.state: Supvisors=test-server.addpipe.com is CHECKING
2022-05-25 16:02:38,531;CRIT;SupervisorListener.on_remote_event: Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/supvisors-0.14-py3.6.egg/supvisors/listener.py", line 285, in on_remote_event
    self.unstack_event(event.data)
  File "/usr/local/lib/python3.6/site-packages/supvisors-0.14-py3.6.egg/supvisors/listener.py", line 297, in unstack_event
    self.supvisors.fsm.on_tick_event(event_identifier, event_data)
  File "/usr/local/lib/python3.6/site-packages/supvisors-0.14-py3.6.egg/supvisors/statemachine.py", line 493, in on_tick_event
    self.context.on_tick_event(identifier, event)
  File "/usr/local/lib/python3.6/site-packages/supvisors-0.14-py3.6.egg/supvisors/context.py", line 429, in on_tick_event
    self.check_instance(status)
  File "/usr/local/lib/python3.6/site-packages/supvisors-0.14-py3.6.egg/supvisors/context.py", line 632, in check_instance
    self.supvisors.zmq.pusher.send_check_instance(status.identifier)
AttributeError: 'NoneType' object has no attribute 'pusher'
```

**Comment by remusnegrota** [comment]
Thank you for looking into this, here is the zipped logs and conf: [supervisor conf and logs.zip](https://github.com/julien6387/supvisors/files/8772701/supervisor.conf.and.logs.zip)

**Comment by julien6387** [comment]
All right. In the Supvisors documentation, I have written that Supervisor must be configured with an internet socket (inet_http_server), which you did in your example. And I indeed never wrote that Supvisors was not compatible with UNIX sockets (unix_http_server).
Actually, I had never considered the possibility that both internet and UNIX sockets could be used in the same Supervisor configuration.
Having tried your example, it appears that the Supvisors plugin is instantiated twice by Supervisor ; I suspect once for the UNIX socket and once for the internet socket.
The Supvisors plugin is clearly not designed to be started twice in the same Supervisor daemon. The second start induces zmq errors.

So to make your example operational, you need to:
* comment the unix_http_server section
* remove the 127.0.0.1 from the inet_http_server configuration
```
[inet_http_server] 
port=:9001
```

I have at least to make it clear in the documentation that Supvisors is not compatible with the use of UNIX sockets with the current design. I could also make a fix to allow the option 'port=127.0.0.1:60000' in the inet_http_server section.

If having a UNIX socket is important for you, I could try to find a way to ensure that the Supvisors plugin is started only once and only in the context of the internet socket. Having Supvisors working with a UNIX socket is quite unlikely.
Please let me know if it solves your issue and/or if you need a fix.

**Comment by remusnegrota** [comment]
Yes, changed the conf as you suggested by removing the [unix_http_server] section and only specifying the port in the [inet_http_server] section fixed the issue, thanks.

I don't have a use case for also needing a UNIX socket right now, but if there is a way to do it, maybe others might find it helpful.
